charts/rancher-istio/1.5.902/questions.yaml

```diff
@@ -1,3 +1,3 @@
 labels:
   rancher.istio.v1.5.902: 1.5.9
-rancher_min_version: 2.4.5-rc1
+rancher_min_version: 2.5.8-rc1
```

Comment:
This could be 2.6, but is actually fine for now since we may do 2.5.x anyway

Reply:
@cbron does that mean I Should set the max version on the 1.5.901chart to 1.5.10?

Reply:
For now just leave it as-is, because it needs to be available in 2.6. Until we make 1.5.902 in 2.5.